work9/index.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+const ul = document.getElementById("js-list");
+const data = [{
+  to: "bookmark.html",
+  img: "1.png",
+  alt: "画像1",
+  text: "ブックマーク",
+},
+{
+  to: "message.html",
+  img: "2.png",
+  alt: "画像2",
+  text: "メッセージ"
+},
+]
+
+function getData() {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve(data);
+
+      reject(new Error("エラー"));
```

Comment:
why exist resolve and reject together. ??

Reply:
It's my misunderstanding
"Resolve and reject are used as a set" is generally used in conditional branching such as if statements.
You shouldn't have rejected and resolved in the same scope.

Reply:
give me fixed commit.

Reply:
excuse me
Here is the modified commit


20b1fa7